contents/handbook/help.mdx

```diff
@@ -0,0 +1,122 @@
+* [Jams](/jams) wore a skin tight green all-body suit for months to improve his Zoom background game without us realizing.
+* <Emoji name="ben-peace" src="/images/emojis/ben-peace.png" /> Ben has the same pose in 90% of PostHog photos. It's a reference to a meme.
+* <Emoji name="hype-X" src="/images/emojis/lottie-hype.gif" /> where X is a team member. Used in times of extremely impressive performance, unless used sarcastically.
+* Mr Blobby. We once changed how we ingest session recording data, to use s3 blob storage. We called it Mr Blobby. [Mr Blobby](https://en.wikipedia.org/wiki/Mr_Blobby) is a creepy '90s TV character from the UK. This project was nightmarishly hard, which is why this character was fitting.
+* Paul will make you eat Gelato at every offsite.
+* Sometimes people screenshot each other's faces and zoom screens and use them as their backgrounds. Usually when an all hands is too dry.
+* Charles wore a suit to his performance review. He is the only person in history to wear a suit to anything PostHog related. Unsure if he was making a point, we later abandoned the practice of performance reviews regardless.
+* We took lots of buses at an offsite in Portugal. The roads were incredibly twisty, the driver was in a bad mood, drove too quickly, and people threw up. It was bad.
+* <Emoji name="sparksjoy" src="/images/emojis/sparksjoy.png" /> / <Emoji name="does_not_spark_joy" src="/images/emojis/does_not_spark_joy.png" /> A reference to <a href="https://konmari.com/marie-kondo-rules-of-tidying-sparks-joy/">Marie Kondo's book</a> on tidying your house, generally used to describe things that are particularly good or bad from a user's perspective
+* <Emoji name="eu-thumbsup" src="/images/emojis/eu-thumbsup.png" /> / <Emoji name="thumbs-down-eu" src="/images/emojis/thumbs-down-eu.png" /> We once made <a href="https://www.isgoogleanalyticsillegal.com">isgoogleanalyticsillegal.com</a> when there were privacy rulings about Google Analytics. We put it on HN, got the top of the front page and it was our biggest <em>ever</em> day of signups at the time. The website was supposed to be tongue in cheek, but the internet took it seriously. The person in the emoji is Ursula von der Leyen who introduced the GDPR legislation.
+* IPO promises. There is a list of these that is brought out at certain moments. You may see.
+* [Marius](/community/profiles/1) will train you on Post-it notes if you go to an offsite with him. Success of a good Post-it note posting is in the lift away from the surface - the most important thing is to peel, as opposed to pulling, the Post-it note off.
+* Three finger rule - another Marius invention, if someone holds up three fingers while you're talking, it means you aren't being concise enough. We don't actually use this much as it's predictably awkward and distracting, so ruins any meeting it could have otherwise helped.
```

Comment:
I think the "How you can help" page should end with "we're hiring, come join us!" as a sign off

Reply:
done